maven-plugin/src/main/java/com/github/swissquote/carnotzet/maven/plugin/AbstractZetMojo.java

```diff
@@ -1,30 +1,36 @@
 package com.github.swissquote.carnotzet.maven.plugin;
 
-import java.nio.file.Paths;
-
-import org.apache.maven.execution.MavenSession;
-import org.apache.maven.plugin.AbstractMojo;
-import org.apache.maven.plugin.MojoExecutionException;
-import org.apache.maven.plugin.MojoFailureException;
-import org.apache.maven.plugins.annotations.Component;
-import org.apache.maven.plugins.annotations.Parameter;
-import org.apache.maven.project.MavenProject;
-import org.apache.maven.project.ProjectBuilder;
-import org.apache.maven.settings.Settings;
-import org.slf4j.bridge.SLF4JBridgeHandler;
-
 import com.github.swissquote.carnotzet.core.Carnotzet;
 import com.github.swissquote.carnotzet.core.CarnotzetConfig;
+import com.github.swissquote.carnotzet.core.CarnotzetExtension;
+import com.github.swissquote.carnotzet.core.CarnotzetExtensionsFactory;
 import com.github.swissquote.carnotzet.core.maven.CarnotzetModuleCoordinates;
 import com.github.swissquote.carnotzet.core.runtime.api.ContainerOrchestrationRuntime;
 import com.github.swissquote.carnotzet.core.runtime.log.LogListener;
 import com.github.swissquote.carnotzet.core.runtime.log.StdOutLogPrinter;
 import com.github.swissquote.carnotzet.maven.plugin.impl.Utils;
 import com.github.swissquote.carnotzet.runtime.docker.compose.DockerComposeRuntime;
-
 import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
 import lombok.Getter;
 import lombok.Setter;
+import org.apache.maven.execution.MavenSession;
+import org.apache.maven.plugin.AbstractMojo;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugin.MojoFailureException;
+import org.apache.maven.plugins.annotations.Component;
+import org.apache.maven.plugins.annotations.Parameter;
+import org.apache.maven.project.MavenProject;
+import org.apache.maven.project.ProjectBuilder;
+import org.apache.maven.settings.Settings;
+import org.slf4j.bridge.SLF4JBridgeHandler;
+
+import java.nio.file.Paths;
+import java.util.Properties;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.ServiceLoader;
+import java.util.Optional;
+import java.util.stream.Collectors;
```

Comment:
Please use the eclipse/intellij code style provided in code_style if you want to use your IDE import management option. (we should add some checkstyle rules)